### ISSUE TITLE
Keys with '.' in their name print no output

### ISSUE THREAD

**Issue by hasit** [hasit/bolter#8]
For example,

```
$ bolter -f emails.db -b blah@blah.com
```

 prints nothing because we are looking for nested buckets after '.'.

**Comment by nouney** [comment]
Just a thought: Something like `parent->child`

`->` is less likely to occur than `.`

**Comment by hasit** [comment]
@nouney 

That makes sense but would still cause an error for keys with '->' in them. The point is to completely eliminate this bug for all current and future cases instead of just making a hack around it.

Here's another thought:

We can start an interactive shell and `bolter` act like a command line shell `cd` tool. Users can go in and out of files, buckets, and nested buckets. https://github.com/abiosoft/ishell looks promising.

**Comment by ross-spencer** [comment]
Hey Hasit - I found your tool useful for looking at my data structures but needed a more interactive mode for deeper nested buckets. I've made a version here that's interactive: https://github.com/kval-access-language/bolter

I'm not sure it's suitable for a pull request because it's an extensive change including:

- Removed machine readable output
- Implemented my own Bolt query mechanism to simplify the code

But any opinion is useful.

Also, what's the LICENSE of this work, as I'd like to make sure I comply for any further changes if I use it from my fork. Cheers.

**Comment by hasit** [comment]
@ross-spencer 

Hello. I am glad this tool proved helpful for you. I did get a chance to look over your changes, but not in much detail. Your changes address some of the concerns that I have raised in this issue. I would be happy to review a PR from you. If we do decide to merge your changes, @nouney or I could look into implementing machine readable output functionality in a separate branch (which, I think, shouldn't be much extra work).

Thank you. I look forward to your PR.
